torch/nested/_internal/nested_tensor.py

```diff
@@ -232,6 +240,17 @@ def backward(ctx, gO: NestedTensor):  # type: ignore[override]
         return gO.values(), None, None
 
 
+# Not actually a view!
+class ViewNoncontiguousNestedFromBuffer(torch.autograd.Function):
```

Comment:
nit:
```suggestion
class ViewNonContiguousNestedFromBuffer(torch.autograd.Function):
```
is slightly more readable to me at a glance